app/frontend/src/store/modules/form.js

```diff
@@ -13,7 +13,7 @@ const genInitialForm = () => ({
   sendSubRecieviedEmail: false,
   showSubmissionConfirmation: true,
   submissionReceivedEmails: [],
-  userType: ''
+  userType: IdentityMode.TEAM
```

Comment:
I can't remember but didn't we originally have this set to Team by default, then changed it to be empty by default so that it wouldn't populate incorrectly or something?

Reply:
Yes but looks like the refactor (the same one that lead to not needing the fetch in FormSettings) has fixed that. I regression tested creating forms, modifying existing settings, and permissions on form load and it's all good with the default set here now.